core-ui/src/main/java/com/anytypeio/anytype/core_ui/features/editor/holders/relations/RelationBlockViewHolder.kt

```diff
@@ -169,8 +170,13 @@ sealed class RelationBlockViewHolder(
 
         override fun applyRelationValue(item: ObjectRelationView) {
             tvValue.apply {
-                text = item.value
                 if (item is ObjectRelationView.Default) {
+                    if (item.format == Relation.Format.NUMBER && item.key == "sizeInBytes") {
```

Comment:
It’s better not to use plane Strings in classes; instead, use const val. Here, it should be Relations.SIZE_IN_BYTES.

Reply:
Also, let’s use formatting only for objects with layouts: `SupportedLayouts.fileLayouts`.